typescript-selenium/pageobjects/dashboard/Workspaces.ts

```diff
@@ -0,0 +1,95 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { TestConstants } from "../../TestConstants";
+import { injectable, inject } from "inversify";
+import { DriverHelper } from "../../utils/DriverHelper";
+import { CLASSES } from "../../inversify.types";
+import { By } from "selenium-webdriver";
+
+
+@injectable()
+export class Workspaces {
+    private readonly driverHelper: DriverHelper;
+    private static readonly TITLE: string = ".che-toolbar-title-label";
+    private static readonly ADD_WORKSPACE_BUTTON_CSS: string = "#add-item-button";
+    private static readonly START_STOP_WORKSPACE_TIMEOUT: number = TestConstants.TS_SELENIUM_START_WORKSPACE_TIMEOUT
+
+    constructor(
+        @inject(CLASSES.DriverHelper) driverHelper: DriverHelper
+    ) {
+        this.driverHelper = driverHelper;
```

Comment:
I would recommend to use parameter-properties in the constructor: https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties
```typescript
class A {
    constructor(private readonly driverName: string) {}

    test() {
        console.log(this.driverName);
    }
}
```
And you can avoid a lot of `this.field = field` constructions.